src/styles/global-styles.scss

```diff
@@ -27,3 +27,7 @@ h2 {
 h3 {
   font-size: 1.33rem;
 }
+
+.d-none {
+  display: none !important;
```

Comment:
Please look for a way of not using `!important` here
https://developer.mozilla.org/en-US/docs/Web/CSS/Specificity

Reply:
I removed this css from here(global) and applied in the the navbar.css but it's not working. I wonder why🤔. I tried looking for it's alternative but still no clue yet. Still looking

Reply:
this one needs to be removed